generator_test.go

```diff
@@ -71,6 +72,7 @@ var expectedFluentDConfig = `@include /etc/head_file.conf
 
 <source>
     @type forward
+    tag tag
```

Comment:
Tag is technically not used with forward; but this looks good for the purposes of testing